### DISCUSSION TITLE
Login dashboard

### DISCUSSION THREAD

**Discussion by reyptr27** [dewanakl/undangan-api#19] [Q&A] [answered]
Loginnya gimana ya mas ? Saya pakai user@example.com dapat error "unexpected token ... is not valid JSON"

**Comment by dewanakl** [comment, marked as answer]
Boleh sertakan ss nya mas
pastikan sudah deploy dengan benar

**Reply by reyptr27** [reply]
sudah solved mas, saya belum ganti data url di dashboard.html nya.
terimakasih sudah merespon. btw undangan yang bagus mas sangat membantu.

**Reply by dewanakl** [reply]
Sama sama mas
